### ISSUE TITLE
Cross Browser Issue on Vue API Docs Sidebar

### ISSUE THREAD

**Issue by alexhernandez** [vuejs/vuejs.org#598]
Chrome (Version 54.0.2840.98 (64-bit)):
![chrome](https://cloud.githubusercontent.com/assets/2424355/20466800/e3b920f4-af2e-11e6-8e6d-84fe6a5567a6.png)

Safari (Version 9.1.2 (11601.7.7)):
![safari](https://cloud.githubusercontent.com/assets/2424355/20466801/e3b9ba50-af2e-11e6-9bb4-8978c5cb0546.png)
![safari-errors](https://cloud.githubusercontent.com/assets/2424355/20466822/34fe2bee-af2f-11e6-8446-4664562ad394.png)

Console:
```
[Warning] Unexpected CSS token: $ (page.css, line 305)
[Error] SyntaxError: Unexpected identifier 'i'. Expected either 'in' or 'of' in enumeration syntax.
	(anonymous function) (common.js:61)
[Error] Failed to load resource: the server responded with a status of 404 (HTTP/2.0 404) (docsearch.min.js.map, line 0)
```

Common.js:
```
    function levenshteinDistance (a, b) {
      var m = []
      if (!(a && b)) return (b || a).length
      for (let i = 0; i <= b.length; m[i] = [i++]) {} <----- SyntaxError: Unexpected identifier 'i'. Expected either 'in' or 'of' in enumeration syntax.
      for (let j = 0; j <= a.length; m[0][j] = j++) {}
      for (let i = 1; i <= b.length; i++) {
        for (let j = 1; j <= a.length; j++) {
          m[i][j] = b.charAt(i - 1) === a.charAt(j - 1)
            ? m[i - 1][j - 1]
            : m[i][j] = Math.min(
              m[i - 1][j - 1] + 1,
              Math.min(m[i][j - 1] + 1, m[i - 1][j] + 1))
        }
      }
      return m[b.length][a.length]
    }
```

**Comment by kazupon** [comment]
This issue is same as #595

I fixed this issue and send #597

**Comment by yyx990803** [comment]
Should be fixed now.
